Microsoft.Identity.Web/Resource/ScopesRequiredByWebAPIExtension.cs

```diff
@@ -44,7 +20,7 @@ public static class ScopesRequiredByWebAPIExtension
         /// <param name="acceptedScopes">Scopes accepted by this API</param>
         /// <exception cref="HttpRequestException"/> with a <see cref="HttpResponse.StatusCode"/> set to 
         /// <see cref="HttpStatusCode.Unauthorized"/>
-        public static void VerifyUserHasAnyAcceptedScope(this HttpContext context, params string[] acceptedScopes)
+        public static void VerifyUserHasAcceptedScope(this HttpContext context, params string[] acceptedScopes)
```

Comment:
this is an argument with a params modifier.
In the case of this sample, it's used with one scope, but it can be used with several? 

think of an action which could be accepted if the calling application has Api.Read or Api.Write?